### ISSUE TITLE
Clean occupation data

### ISSUE THREAD

**Issue by SimonLab** [TheScienceMuseum/collectionsonline#518]
```
"occupation": [
" computer pioneer",
"mathematician",
"mathematician; computer pioneer"
],
```
- [ ] Delete the last value
- [ ] Delete unnecessary spaces in values

**Comment by jamieu** [comment]
This should now be correct in the index, @SimonLab do we need to revert anything?

**Comment by SimonLab** [comment]
@jamieu yes we might delete some small changes but it should be quick, I'm going to have a look now

**Comment by SimonLab** [comment]
@jamieu it doesn't seem that the changes have been applied to the index yet. I've just check on ada (agent/cp38764/) and the spaces and last values is still here:
![occupation-people](https://cloud.githubusercontent.com/assets/6057298/20679180/bd6463b4-b591-11e6-9f55-6ca88cba7f3a.png)

**Comment by SimonLab** [comment]
This is now resolved in the index, see #773
